packages/devtools_app/test/eval_field_test.dart

```diff
@@ -0,0 +1,251 @@
+import 'package:devtools_app/devtools_app.dart';
```

Comment:
this file needs a copy right. You can copy from another test file and paste here, just make sure the year is 2022. Also, please move this test to the `test/debugger/` directory

Reply:
There is no `test/debugger/` directory, or do you mean that the test file should be prefixed with `debugger_`?

Reply:
I see - Added the copyright and moved the file